addons/binding/org.openhab.binding.cbus/src/main/java/org/openhab/binding/cbus/internal/cgate/CGateConnectException.java

```diff
@@ -0,0 +1,30 @@
+/**
+ *  CGateInterface - A library to allow interaction with Clipsal C-Gate.
```

Comment:
Hey @kaikreuzer 

I hope I'm not out of line, but, I thought I bring this community and contribution to Dave Oxley's attention.  Scott hasn't replied in a while and I thought I'd try and get the ball rolling on this great contribution while we wait for his return.

Clipsal's C-Cbus has been around for a very long time here in Australia and I know that there is a large existing installation base.  It would be amazing to bring openHAB to this userbase.  I hope Scott comes back soon and finds the time to make the requested amendments.